src/node.h

```diff
@@ -209,7 +209,8 @@ NODE_EXTERN void FreeEnvironment(Environment* env);
 
 NODE_EXTERN void EmitBeforeExit(Environment* env);
 NODE_EXTERN int EmitExit(Environment* env);
-NODE_EXTERN void RunAtExit(Environment* env);
+NODE_DEPRECATED("Use Environment::RunAtExit()",
+                NODE_EXTERN void RunAtExit(Environment* env));
```

Comment:
Oh, one more thing: `env.h` is not a public header, so if anyone actually wants to use `RunAtExit` in an addon, deprecating this puts them in quite the difficult position. ;)